pkg/controllers/allocation/scheduling/topologygroup.go

```diff
@@ -53,7 +53,7 @@ func (t *TopologyGroup) Increment(domain string) {
 // NextDomain chooses a domain that minimizes skew and increments its count
 func (t *TopologyGroup) NextDomain() string {
 	minDomain := ""
-	minCount := math.MaxInt64
+	minCount := math.MaxInt32
```

Comment:
Is there a reason we can't do 64 here?

Reply:
Yeah -- one of our target releases doesn't have int64 =/.